scripts/travis_publish.sh

```diff
@@ -10,8 +10,9 @@ else
 fi
 
 COMMIT_MESSAGE=$(git show -s --format=%B $TRAVIS_COMMIT | tr -d '\n')
+PACKAGE_JSON_VERSION=$(node ./scripts/package-version.js)
```

Comment:
No problem the way you have it, but FYI I've just done `node -e "console.log(require('./package.json').version)"` in the past without issue.